crates/ethereum/payload/src/lib.rs

```diff
@@ -438,13 +439,17 @@ where
             .map_err(PayloadBuilderError::other)?;
 
         excess_blob_gas = if chain_spec.is_cancun_active_at_timestamp(parent_header.timestamp) {
-            let parent_excess_blob_gas = parent_header.excess_blob_gas.unwrap_or_default();
-            let parent_blob_gas_used = parent_header.blob_gas_used.unwrap_or_default();
-            Some(calc_excess_blob_gas(parent_excess_blob_gas, parent_blob_gas_used))
+            let blob_params = if chain_spec.is_prague_active_at_timestamp(parent_header.timestamp) {
```

Comment:
shouldn't we check payload timestamp here instead?

Reply:
excess is derived from the parent